.github/workflows/release.yml

```
@@ -41,11 +41,23 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
 
+      - name: Check if NPM_TOKEN is set
+        id: check_token
+        run: |
+          if [ -z "${{ secrets.NPM_TOKEN }}" ]; then
+            echo "NPM_TOKEN is not set. Is it set it in your repository secrets?"
+            exit 1
+          fi
+        shell: bash
+
```

```suggestion
      - name: Check if NPM_TOKEN is set
        id: check_token
        run: |
          if [ -z "${{ secrets.NPM_TOKEN }}" ]; then
            echo "NPM_TOKEN is not set. Is it set it in your repository secrets?"
            exit 1
          fi
        shell: bash
```
Could get rid of this but meh, let's keep it